adapter/memQuota/memQuota.go

```diff
@@ -73,7 +73,11 @@ func (builder) ValidateConfig(cfg adapter.Config) (ce *adapter.ConfigErrors) {
 	return
 }
 
-func (builder) NewQuotasAspect(env adapter.Env, c adapter.Config, d map[string]*adapter.QuotaDefinition) (adapter.QuotasAspect, error) {
+func (builder) Configure(quotas map[string]*adapter.QuotaDefinition) error {
```

Comment:
Do we need to do this change ? This interface does not represent the eventual Builder interface for the template based adapters. We will end of moving this configure method into Builder.Build method eventually when porting these adapters to the new world. If we can can somehow have this PR not touch any of the existing adapters, it will be ideal. Unless it is really needed.